radixdlt-core/radixdlt/src/main/java/com/radixdlt/store/berkeley/BerkeleyLedgerEntryStore.java

```diff
@@ -310,11 +310,15 @@ public ByteBuffer verifyVirtualSubstate(SubstateId substateId)
 					return BerkeleyLedgerEntryStore.this.loadAddr(dbTxn, addr);
 				}
 			});
+			txnLog.flush();
 			dbTxn.commit();
 			return result;
-		} catch (Exception e) {
+		} catch (RadixEngineException e) {
```

Comment:
Hrm, having RadixEngineException here looks wrong.

Reply:
yeah, my bad, I've misread the code